### ISSUE TITLE
Docker version doesn't start

### ISSUE THREAD

**Issue by matte0080** [escomputers/BASD#49]
Hi, 

when i try yo start docker version with command:

`docker run -e DJANGO_SECRET_KEY='Prova1234' -dp 8000:8000 escomputers/basd`

I receve first this warning:

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
60981c954b455beac87611b5a2aded985fb6dce41410266a84e0c394806049f7
```

then inside container log i see:

```
2023-01-30 09:21:25 Unknown command: 'makemigrations\r'. Did you mean makemigrations?
2023-01-30 09:21:25 Type 'manage.py help' for usage.
: not found09:21:25 entrypoint.sh: 2: entrypoint.sh: 
2023-01-30 09:21:26 Unknown command: 'migrate\r'. Did you mean migrate?
2023-01-30 09:21:26 Type 'manage.py help' for usage.
: not found09:21:26 entrypoint.sh: 4: entrypoint.sh: 
2023-01-30 09:21:28 Performing system checks...
2023-01-30 09:21:28 
2023-01-30 09:21:29 System check identified no issues (0 silenced).
2023-01-30 09:21:29 January 30, 2023 - 08:21:29
2023-01-30 09:21:29 Django version 3.2.16, using settings 'BASD.settings'
2023-01-30 09:21:29 Starting development server at http://0.0.0.0:8000/
2023-01-30 09:21:29 Quit the server with CONTROL-C.
2023-01-30 09:22:44 [30/Jan/2023 08:22:44] "GET / HTTP/1.1" 400 143

```

But when i go to localhost:8000 i see 400 Bad Request error.

**Comment by matte0080** [comment]
I retry with 127.0.0.1:8000 and something appears but there are many errors for static bundle. So the UI doesn't works.

```
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET / HTTP/1.1" 200 19416
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/jquery/jquery.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/moment/moment-with-locales.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/jquery-validation/jquery.validate.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/jquery-validation/additional-methods.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/bootstrap-datetimepicker/css/bootstrap-datetimepicker.min.css HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/moment/moment-timezone-with-data-10-year-range.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/select2/css/select2.min.css HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/select2/css/select2-bootstrap4.min.css HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/bootstrap-datetimepicker/css/bootstrap-datetimepicker.min.css HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/select2/js/select2.full.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/github/buttons.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/bootstrap-datetimepicker/js/bootstrap-datetimepicker.min.js HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/select2/css/select2.min.css HTTP/1.1" 404 179
2023-01-30 09:49:38 [30/Jan/2023 08:49:38] "GET /static/plugins/select2/css/select2-bootstrap4.min.css HTTP/1.1" 404 179
```

**Comment by escomputers** [comment]
Hi! You were right, there's was an error related to Django settings.py that does not allow to access web page.
Fixed with last commit a11c50a9b80510c66d7d444cd38752d8cc91e7cd
Delete all images and containers and run again Docker install instructions, if you'll get further problems feel free to raise a new issue.
